task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -41,7 +41,7 @@ def __invariant__(conf):
         return ('image' in conf if conf.containerizer == 'DOCKER' else True,
                 'Image required for chosen containerizer')
 
-    uuid = field(type=uuid.UUID, initial=uuid.uuid4)
+    uuid = field(type=str, initial=str(uuid.uuid4()))
```

Comment:
what's the reason behind this change?

Reply:
uuid for returns an object of type UUID. We cannot append a string to it.

Reply:
right, so it shouldn't be called uuid then